addons/event/models/event_registration.py

```diff
@@ -316,7 +316,9 @@ def _update_mail_schedulers(self):
             # we could simply call _create_missing_mail_registrations and let cron do their job
             # but it currently leads to several delays. We therefore call execute until
             # cron triggers are correctly used
-            onsubscribe_schedulers.with_user(SUPERUSER_ID).execute()
+            onsubscribe_schedulers.with_context(
```

Comment:
can we add a test that check that the other event mail are not sent ?

Reply:
Not sure to understand ? The other types (before event / after event) are triggered by the cron only.

If it is about "existing registrations", there is a test for that :) .

Reply:
> The other types (before event / after event) are triggered by the cron only

Ah True, it's not the same code, ok then